benchmarks/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+#FIXME_OPENMPTARGET - compiling in debug mode causes ICE.
```

Comment:
@rgayatri23 do you want this to be guarded?

Reply:
Yeah that would be a good idea to protect it.